### PR TITLE
Kamino-lending

### PR DESCRIPTION
**Pull request by g1nt0ki**

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/kamino-lending: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:205:11

**Comment by llamatester** [comment]
Error while running adapter at projects/kamino-lending: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:205:11

File: projects/kamino-lending/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+const { PublicKey, Keypair } = require('@solana/web3.js');
+const { getConnection, sumTokens } = require('../helper/solana');
+const { Program } = require('@project-serum/anchor');
+const kaminoIdl = require('./kamino-lending-idl.json');
+const { Token, TOKEN_PROGRAM_ID } = require('@solana/spl-token');
```
Comment (peroxy) [inline projects/kamino-lending/index.js:5]: I think you will have to remove this line as well

Comment (peroxy) [reply projects/kamino-lending/index.js:5]: https://github.com/DefiLlama/DefiLlama-Adapters/pull/8025/commits/e189b8f4f1d380f7cae2497970f713d56861c0bb

Comment (peroxy) [reply projects/kamino-lending/index.js:5]: This is how I did it ^

Comment (g1nt0ki) [reply projects/kamino-lending/index.js]: I think it is an indirect dependency, but changed it to your code :)

**Comment by llamatester** [comment]
Error while running adapter at projects/kamino-lending: 
        
 
  
    
    
    FetchError: request to https://try-rpc.mainnet.solana.blockdaemon.tech/ failed, reason: getaddrinfo ENOTFOUND try-rpc.mainnet.solana.blockdaemon.tech
    Truncated error stack:
        at ClientRequest.<anonymous> (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/node-fetch/lib/index.js:1491:11)
        at ClientRequest.emit (node:events:517:28)
        at TLSSocket.socketErrorListener (node:_http_client:501:9)
        at TLSSocket.emit (node:events:529:35)

**Pull request merged**